src/server/middleware/sendHtml.js

```diff
@@ -57,7 +57,19 @@ export function safeSend(res, ...payload) {
   }
 }
 
-export function renderStaticErrorPage(res) {
+export async function fetchCustomErrorPage(fallbackUrl, res) {
+  const response = await fetch(fallbackUrl);
+  // If the Content-Type is not text/html throw an error
+  const contentType = response.headers.get('content-type');
+  if (!contentType.includes('text/html')) {
+    throw new Error('Content-Type was not of type text/html');
```

Comment:
Is it possible to fallback to the standard error page if something goes wrong fetching / parsing the custom url?

Reply:
@mtomcal said that we should throw here

Reply:
did you test this ? the purpose of this is to catch any errors that have bubbled up. its meant the final safety net. this needs to be bullet proof.

Reply:
I tested with responses of `application/json` and `text/html`. The ones with `application/json` threw and the ones with `text/html` did not and returned that result.